### PR TITLE
Added recipe for Rx

### PR DESCRIPTION
**Pull request by pmlandwehr**
`rx` is needed for a new version of `graphql-core`

### CODE REVIEW

File: recipes/rx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - rx.subjects
+    - rx.testing
+
+about:
+  home: http://reactivex.io
+  license: Apache 2.0
+  license_family: Apache
+  license_file: {{ RECIPE_DIR }}/LICENSE
+  summary: Reactive Extensions (Rx) for Python
+  description: "A library for composing asynchronous and event-based programs using observable collections and LINQ-style query operators in Python."
```
Comment (CJ-Wright) [inline recipes/rx/meta.yaml]: Would it be possible to break this up over multiple lines?

File: recipes/rx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  {{ hash_type }}: {{ hash_value }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  setup:
```
Comment (CJ-Wright) [inline recipes/rx/meta.yaml]: I don't think this is a valid sub-section, you might try build

**Comment by synapticarbors** [comment]
This looks like it could be merged. Any reason to delay? (Just wanted to make sure since I don't like usurping someone else's review).

**Comment by mariusvniekerk** [comment]
@synapticarbors @CJ-Wright this seems done already https://github.com/conda-forge/rx-feedstock

**Pull request closed without merging**